### PR TITLE
Fix video_devices on s390x

### PR DESCRIPTION
**Pull request by smitterl**
On s390x virtio video devices are always translated to virtio-gpu-ccw

Test run on s390x:
```bash
# avocado run --vt-type libvirt --vt-machine-type s390-virtio --vt-connect-uri qemu:///system virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio
JOB ID     : 41678a0f37b76444b13a0d9c7fa7b1da9952362a
JOB LOG    : /root/avocado/job-results/job-2019-10-30T11.10-41678a0/job.log
 (1/3) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio.model_test: PASS (6.11 s)
 (2/3) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio.heads_test.default_heads: PASS (6.65 s)
 (3/3) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio.heads_test.specified_heads: PASS (5.72 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 21.08 s
# avocado run --vt-type libvirt --vt-machine-type s390-virtio --vt-connect-uri qemu:///system virtual_devices.video_devices.positive_test.primary_video.virtio.no_secondary_video
JOB ID     : 1bf39a89d92ffde35d9594a327437241b89bc4da
JOB LOG    : /root/avocado/job-results/job-2019-10-30T11.10-1bf39a8/job.log
 (1/3) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.no_secondary_video.model_test: PASS (5.33 s)
 (2/3) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.no_secondary_video.heads_test.default_heads: PASS (5.73 s)
 (3/3) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.no_secondary_video.heads_test.specified_heads: PASS (5.33 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 18.04 s
```

Test run on x86_64:
```bash
# avocado run --vt-type libvirt --vt-connect-uri qemu:///system virtual_devices.video_devices
JOB ID     : d8691f95ec455a65e6c54c4945e4fa11d7618283
JOB LOG    : /root/avocado/job-results/job-2019-10-30T16.10-d8691f9/job.log
 (01/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.model_test: PASS (4.25 s)
 (02/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.heads_test.default_heads: PASS (4.27 s)
 (03/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.heads_test.specified_heads: PASS (4.17 s)
 (04/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.mem_test.ram.default_size: PASS (4.35 s)
 (05/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.mem_test.ram.specified_size.power_of_2: PASS (4.16 s)
 (06/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.mem_test.ram.specified_size.zero_size: PASS (4.11 s)
 (07/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.mem_test.ram.specified_size.non_power_of_2: PASS (4.80 s)
 (08/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.mem_test.vram.qxl_default_size: PASS (4.76 s)
 (09/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.mem_test.vram.specified_size.power_of_2:PASS (4.25 s)
 (10/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.mem_test.vram.specified_size.qxl_zero_size: PASS (4.55 s)
 (11/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.mem_test.vram.specified_size.non_power_of_2: PASS (4.29 s)
 (12/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.mem_test.vgamem.default_size: PASS (4.51 s)
 (13/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.mem_test.vgamem.power_of_2: PASS (4.25 s)
 (14/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.mem_test.vgamem.zero_size: PASS (4.50 s)
 (15/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.mem_test.vram64.default_size: PASS (4.30 s)
 (16/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.no_secondary_video.mem_test.vram64.power_of_2: PASS (4.35 s)
 (17/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.secondary_video.qxl.model_test: PASS (4.55 s)
 (18/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.secondary_video.qxl.heads_test.default_heads: PASS (4.90 s)
 (19/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.secondary_video.qxl.heads_test.specified_heads: PASS (4.86 s)
 (20/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.secondary_video.virtio.model_test: PASS (4.72 s)
 (21/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.secondary_video.virtio.heads_test.default_heads: PASS (4.82 s)
 (22/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.qxl.secondary_video.virtio.heads_test.specified_heads: PASS (4.63 s)
 (23/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.vga.no_secondary_video.model_test: PASS (4.23 s)
 (24/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.vga.no_secondary_video.heads_test.default_heads: PASS (4.56 s)
 (25/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.vga.no_secondary_video.mem_test.vram.vga_default_size: PASS (4.29 s)
 (26/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.vga.no_secondary_video.mem_test.vram.specified_size.power_of_2:PASS (5.23 s)
 (27/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.vga.no_secondary_video.mem_test.vram.specified_size.vga_zero_size: PASS (4.55 s)
 (28/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.vga.no_secondary_video.mem_test.vram.specified_size.non_power_of_2: PASS (4.29 s)
 (29/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.vga.secondary_video.qxl.model_test: PASS (4.38 s)
 (30/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.vga.secondary_video.qxl.heads_test.default_heads: PASS (4.52 s)
 (31/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.vga.secondary_video.virtio.model_test: PASS (4.59 s)
 (32/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.vga.secondary_video.virtio.heads_test.default_heads: PASS (4.78 s)
 (33/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.cirrus.no_secondary_video.model_test: PASS (4.26 s)
 (34/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.cirrus.no_secondary_video.heads_test.default_heads: PASS (4.50 s)
 (35/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.cirrus.secondary_video.qxl.model_test: PASS (4.66 s)
 (36/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.cirrus.secondary_video.qxl.heads_test.default_heads: PASS (4.70 s)
 (37/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.cirrus.secondary_video.virtio.model_test: PASS (4.80 s)
 (38/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.cirrus.secondary_video.virtio.heads_test.default_heads: PASS (4.52 s)
 (39/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.no_secondary_video.model_test: PASS (4.13 s)
 (40/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.no_secondary_video.heads_test.default_heads: PASS (4.35 s)
 (41/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.no_secondary_video.heads_test.specified_heads: PASS (4.25 s)
 (42/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.qxl.model_test: PASS (4.60 s)
 (43/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.qxl.heads_test.default_heads: PASS (4.48 s)
 (44/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.qxl.heads_test.specified_heads: PASS (4.45 s)
 (45/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio.model_test: PASS (4.58 s)
 (46/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio.heads_test.default_heads: PASS (4.47 s)
 (47/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio.heads_test.specified_heads: PASS (4.40 s)
 (48/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.ram.empty_value: PASS (3.64 s)
 (49/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.ram.lower_than_bounds: PASS (3.49 s)
 (50/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.ram.ram_vram_upper_than_bounds: PASS (3.77 s)
 (51/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.ram.non_numeric_value: PASS (3.51 s)
 (52/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.ram.float_vaule: PASS (3.78 s)
 (53/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vram.empty_value: PASS (3.55 s)
 (54/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vram.lower_than_bounds: PASS (3.64 s)
 (55/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vram.ram_vram_upper_than_bounds: PASS (3.55 s)
 (56/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vram.non_numeric_value: PASS (3.74 s)
 (57/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vram.float_vaule: PASS (3.43 s)
 (58/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vgamem.empty_value: PASS (3.49 s)
 (59/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vgamem.lower_than_bounds: PASS (3.55 s)
 (60/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vgamem.vgamem_vram64_upper_than_bounds: PASS (3.64 s)
 (61/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vgamem.non_numeric_value: PASS (3.43 s)
 (62/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vgamem.non_power_of_2: PASS (3.80 s)
 (63/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vgamem.float_vaule: PASS (3.57 s)
 (64/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vram64.empty_value: PASS (3.68 s)
 (65/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vram64.lower_than_bounds: PASS (3.62 s)
 (66/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vram64.vgamem_vram64_upper_than_bounds: PASS (3.53 s)
 (67/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vram64.non_numeric_value: PASS (3.58 s)
 (68/68) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.negative_test.qxl.mem_test.vram64.float_vaule: PASS (3.60 s)
RESULTS    : PASS 68 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 290.40 s
```

### CODE REVIEW

File: libvirt/tests/src/virtual_device/video_devices.py [hunk 1]
```diff
@@ -112,10 +119,16 @@ def check_heads_test_cmd_line(model_type, is_primary=None, **kwargs):
         """
         cmdline = open('/proc/%s/cmdline' % vm.get_pid()).read().replace("\x00", " ")
         logging.debug("the cmdline is: %s" % cmdline)
+        # s390x only supports virtio
+        guest_arch = params.get("vm_arch_name")
```
Comment (chunfuwen) [inline libvirt/tests/src/virtual_device/video_devices.py]: maybe bring this variable to head of this file as shared variable

Comment (dzhengfy) [reply libvirt/tests/src/virtual_device/video_devices.py]: agree

Comment (smitterl) [reply libvirt/tests/src/virtual_device/video_devices.py]: @chunfuwen @dzhengfy Thanks for pointing that out :) Fixed.

Test run on s390x:
```bash
# avocado run --vt-type libvirt --vt-machine-type s390-virtio --vt-connect-uri qemu:///system virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio
JOB ID     : 18d589a1f461b472e2339082b47b8995e42532ea
JOB LOG    : /root/avocado/job-results/job-2019-11-07T06.08-18d589a/job.log
 (1/3) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio.model_test: PASS (262.71 s)
 (2/3) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio.heads_test.default_heads: PASS (31.78 s)
 (3/3) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio.heads_test.specified_heads: PASS (16.58 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 321.72 s
```

Comment (smitterl) [reply libvirt/tests/src/virtual_device/video_devices.py]: Thansk @chunfuwen @dzhengfy I fixed it.

Test run on s390x:
```bash
# avocado run --vt-type libvirt --vt-machine-type s390-virtio --vt-connect-uri qemu:///system virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio
JOB ID     : 55e6cf30882e8cfa4ea9f94ece44ca6f911a12ee
JOB LOG    : /root/avocado/job-results/job-2019-11-07T06.27-55e6cf3/job.log
 (1/3) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio.model_test: PASS (13.04 s)
 (2/3) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio.heads_test.default_heads: PASS (13.15 s)
 (3/3) type_specific.io-github-autotest-libvirt.virtual_devices.video_devices.positive_test.primary_video.virtio.secondary_video.virtio.heads_test.specified_heads: PASS (20.78 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 51.82 s
```

**Pull request merged**